lua/luasnip/extras/init.lua

```diff
@@ -113,6 +113,30 @@ local function match(index, _match, _then, _else)
 	return F(func, index)
 end
 
+local function jump_to_snip(snip)
```

Comment:
I think this deserves its own module, then we can cleanly provide different heuristics for finding the correct snippet (after making that configurable ofc :D)
For example, finding in snipmate/lua/vscode could be made more precise by writing it specifically for that type of snippet

Reply:
ok. i'll move this into its own module.